shardingsphere-mode/shardingsphere-mode-type/shardingsphere-memory-mode/shardingsphere-memory-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/memory/lock/MemoryLockContext.java

```diff
@@ -56,7 +58,6 @@ public ShardingSphereLock getOrCreateSchemaLock(final String schemaName) {
     
     @Override
     public boolean isLockedSchema(final String schemaName) {
-        ShardingSphereLock shardingSphereLock = locks.get(schemaName);
-        return null != shardingSphereLock && shardingSphereLock.isLocked(schemaName);
+        return null != schemaName && null != locks.get(schemaName) && locks.get(schemaName).isLocked(schemaName);
     }
```

Comment:
`locks.get(schemaName)` run 2 times, the second time of `locks.get(schemaName)` might return different value as the first result.

Reply:
I will fix.